scrabble.js

```diff
@@ -1,16 +1,202 @@
 
 const Scrabble = {
+
+
   score(word) {
 
+    let wordScore = 0
+
+    const SCORING_RUBRIK = {
+      'a': 1,
+      'e': 1,
+      'i': 1,
+      'o': 1,
+      'u': 1,
+      'l': 1,
+      'n': 1,
+      'r': 1,
+      's': 1,
+      't': 1,
+      'd': 2,
+      'g': 2,
+      'b': 3,
+      'c': 3,
+      'm': 3,
+      'p': 3,
+      'f': 4,
+      'h': 4,
+      'v': 4,
+      'w': 4,
+      'y': 4,
+      'k': 5,
+      'j': 8,
+      'x': 8,
+      'q': 10,
+      'z': 10
+    }
+    if (typeof word == 'number'){
+      throw 'Letters only please.';
+    }
+
+    for (let i = 0; i < word.length; i++) {
+
+      let letter = word[i];
+
+      if (!RegExp(/^[a-zA-Z]+$/).test(letter)){
+
+        throw `Letters only please. Letter: ${letter} i == ${i} word: ${word}`
+      }
+
+      wordScore += SCORING_RUBRIK[letter.toLowerCase()];
+    }
+
+    if (word.length === 7){
+      wordScore += 50;
+    }else if (word.length > 7){
+      throw "Words can not be longer than 7 letters.";
+    }else if (word.length === 0){
+      throw "Please submit a word for the game.";
+    }
+
+    return wordScore;
   },
+
+
   highestScoreFrom(arrayOfWords) {
+    let highWord = ''
+    if (arrayOfWords.length === 0){
+      throw "Empty Array of words to choose the highest score from.";
+    }else if (arrayOfWords.length === 1){
+      highWord = arrayOfWords[0];
+    }else {
+      let highScore = -1
+      for (let i = 0; i < arrayOfWords.length; i++){
+
+        let compareWord = arrayOfWords[i]
+        let score = this.score(compareWord)
+        if (score > highScore){
+          highWord = compareWord
+          highScore = score
+        }else if (score == highScore){
+          highWord =   Scrabble.tiebreaker(compareWord,highWord)
+        }
+      }
+    }
+    return highWord;
+  },
+
+  highestWordScoreWord(arrayOfWords) {
+    let highWord = ''
+    let highScore = -1
+    if (arrayOfWords.length === 0){
```

Comment:
You've got almost exactly the same logic duplicated between here and `highestScoreFrom`. Could you DRY these up somehow?